documentation/src/main/asciidoc/user_guide/counters.adoc

```diff
@@ -157,46 +162,56 @@ EmbeddedCacheManager manager = ...;
 CounterManager counterManager = EmbeddedCounterManagerFactory.asCounterManager(manager);
 ----
 
+==== Remove a counter via CounterManager
+
+WARNING: use with caution.
+
+There is a difference between remove a counter via the `Strong/WeakCounter` interfaces and the `CounterManager`.
+The `CounterManager.remove(String)` removes the counter value from the cluster and removes all the listeners registered
+in the counter in the local counter instance.
+In addition, the counter instance is no longer reusable and it may return an invalid results.
+
+NOTE: The counter is re-created if it is accessed after a removal.
+
 === The Counter
 
 A counter can be strong (`StrongCounter`) or weakly consistent (``WeakCounter``) and both is identified by a name.
 They have a specific interface but they share some logic, namely, both of them are asynchronous
 ( a `CompletableFuture` is returned by each operation), provide an update event and can be reset to its initial value.
 
+If you don't want to use the async API, there is a sync decorator that can be used.
+These are the `SyncStrongCounter` and the `SyncWeakCounter`.
```

Comment:
For discoverability it would be nice to add `SyncStrongCounter sync()` method to `StrongCounter`. Could be even default.

Reply:
I'll have to move the `SyncStrongCounter` to commons. I guess it is ok.